contracts/pool/pools/TokenLockingPoolBase.sol

```diff
@@ -18,10 +16,12 @@ abstract contract TokenLockingPoolBase is
     OwnableUpgradeable,
     ReentrancyGuardUpgradeable
 {
+    error InsufficientAmount();
```

Comment:
Mozes li prebacit ovaj error ispod eventa, cisto da postujemo [solidity style guide](https://docs.soliditylang.org/en/latest/style-guide.html#order-of-layout)